service-commands/src/createUserWithTrackUpload.js

```diff
@@ -0,0 +1,294 @@
+const { _ } = require('lodash')
+const fs = require('fs-extra')
+const path = require('path')
+const fetch = require('node-fetch')
+const assert = require('assert')
+const util = require('util')
+const streamPipeline = util.promisify(require('stream').pipeline)
+
+const AudiusLibs = require('@audius/libs')
+const CreatorNode = require('@audius/libs/src/services/creatorNode')
+
+const ethContractsConfig = require('../../libs/eth-contracts/config.json')
+const dataContractsConfig = require('../../libs/data-contracts/config.json')
+const ETH_PROVIDER_ENDPOINT = 'http://localhost:8546'
+const DISCOVERY_NODE_ENDPOINT = 'http://audius-disc-prov_web-server_1:5000'
+const DATA_CONTRACTS_PROVIDER_ENDPOINT = 'http://localhost:8545'
+const USER_METADATA_ENDPOINT = 'http://cn-um_creator-node_1:4099'
+const IDENTITY_SERVICE_ENDPOINT = 'http://localhost:7000'
+const ETH_REGISTRY_ADDRESS = ethContractsConfig.registryAddress
+const ETH_TOKEN_ADDRESS = ethContractsConfig.audiusTokenAddress
+const ETH_OWNER_WALLET = ethContractsConfig.ownerWallet
+const DATA_CONTRACTS_REGISTRY_ADDRESS = dataContractsConfig.registryAddress
+const TRACK_URLS = [
+  'https://royalty-free-content.s3-us-west-2.amazonaws.com/audio/Gipsy.mp3',
+  'https://royalty-free-content.s3-us-west-2.amazonaws.com/audio/First+Rain.mp3',
+  'https://royalty-free-content.s3-us-west-2.amazonaws.com/audio/Miracle.mp3',
+  'https://royalty-free-content.s3-us-west-2.amazonaws.com/audio/Ice+Cream.mp3',
+  'https://royalty-free-content.s3-us-west-2.amazonaws.com/audio/Street+Tables+Cafe.mp3',
+  'https://royalty-free-content.s3-us-west-2.amazonaws.com/audio/Cowboy+Tears.mp3',
+  'https://royalty-free-content.s3-us-west-2.amazonaws.com/audio/Happy.mp3'
+]
+
+const explicitPrimary = 'http://cn1_creator-node_1:4000'
+let userMetadata = {}
+let trackMetadata = {}
+let trackPath = ''
+
+let AudiusLibsInstance
+
+// =~*=~*=~*=~*=~*=~*=~*=~*=~*=~* START UTIL =~*=~*=~*=~*=~*=~*=~*=~*=~*=~*
+
+/**
+ * Waits input ms
+ * @param {number} milliseconds
+ * @returns
+ */
+const wait = async milliseconds => {
+  console.log(`Waiting ${milliseconds}ms...`)
+  return new Promise(resolve => setTimeout(resolve, milliseconds))
+}
+
+/**
+ * Genererates a random string of uppercase + lowercase chars, optionally with numbers.
+ * @param {number} length
+ * @param {boolean} withNumbers
+ */
+const genRandomString = (length, withNumbers = false) => {
+  const lower = 'abcdefghijklmnopqrstuvwxyz'
+  const upper = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+  const numbers = '0123456789'
+
+  const combined = (lower + upper + (withNumbers ? numbers : '')).split('')
+  return _.range(length)
+    .map(x => _.sample(combined))
+    .join('')
+}
+
+const r6 = (withNum = false) => genRandomString(6, withNum)
+
+/**
+ * Generates a single random user.
+ */
+const getRandomUser = () => {
+  return {
+    name: `name_${r6()}`,
+    email: `email_${r6()}@audius.co`,
```

Comment:
if we could make this more meaningful this would be great. like `srn_{{sha of url}}_date@audius.co`

Reply:
if we have the spId of the service in sauron, lets use that here like `srn_{{spId}}_{{date}}`@audius.co` and similar for handle